src/adapters/mc/asb/CMakeLists.txt

```diff
@@ -23,15 +23,15 @@ set_target_properties(OsConfigResourceAsb PROPERTIES OUTPUT_NAME libOsConfigReso
 
 # Create the policy artifacts ZIP package for the Azure Security Baseline for Linux
 
-add_custom_target(stage_create_zip
+add_custom_target(stage_create_zip_asb
     COMMAND ${CMAKE_COMMAND} -E make_directory ${PROJECT_BINARY_DIR}/Staging
     COMMAND ${CMAKE_COMMAND} -E copy_if_different "${CMAKE_CURRENT_SOURCE_DIR}/AzureLinuxBaseline.metaconfig.json" ${PROJECT_BINARY_DIR}/Staging/
     COMMAND ${CMAKE_COMMAND} -E copy_if_different "${CMAKE_CURRENT_SOURCE_DIR}/AzureLinuxBaseline.mof" ${PROJECT_BINARY_DIR}/Staging/
     COMMAND ${CMAKE_COMMAND} -E copy_if_different $<TARGET_FILE:OsConfigResourceAsb> ${PROJECT_BINARY_DIR}/Staging/Modules/DscNativeResources/OsConfigResource/libOsConfigResource.so
     DEPENDS OsConfigResourceAsb)
 
-add_custom_target(create_zip ALL
+add_custom_target(create_zip_asb ALL
     BYPRODUCTS ${OsConfigRootBinaryDir}/AzureLinuxBaseline.zip
     COMMAND ${CMAKE_COMMAND} -E tar "cfv" "${OsConfigRootBinaryDir}/AzureLinuxBaseline.zip" --format=zip .
-    DEPENDS stage_create_zip
+    DEPENDS stage_create_zip_asb
```

Comment:
>stage_create_zip_asb [](http://example.com/codeflow?start=12&length=20)

nit: stage_create_asb_zip looks better (zip kept at the end as it what's gets created)